app/lib/frontend/templates/layout.dart

```diff
@@ -187,6 +188,8 @@ String _renderSearchBanner({
     'include_discontinued': searchForm?.includeDiscontinued ?? false,
     'include_unlisted': searchForm?.includeUnlisted ?? false,
     'legacy_search_enabled': searchForm?.includeLegacy ?? false,
+    'prerelease_null_safe': requestContext.isNullSafetyDisplayed &&
```

Comment:
`requestContext.isNullSafetyDisplayed` is still behind experimental flag, right?

Reply:
yes, it is